### ISSUE TITLE
plugin-merge-duplicate-imports false positive for `import type`

### ISSUE THREAD

**Issue by EvgenyOrekhov** [coderaiser/putout#71]
```ts
import type { QueryFunctionContext, QueryKey } from "react-query";
import { useInfiniteQuery } from "react-query";
```

Gives the following error: `Duplicate imports should be avoided`.

**Comment by coderaiser** [comment]
Thanks you! Fixed with 1f0a1e7f. Landed in `@putout/plugin-merge-duplicate-imports@3.2.0` 🎉 . Is it works for you?

**Comment by EvgenyOrekhov** [comment]
It works now, thank you!
